packages/voting-contract/src/multitest/contracts/voting.rs

```diff
@@ -37,6 +38,19 @@ pub enum ExecuteMsg {
     },
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum Proposal {
+    DoTheThing {},
```

Comment:
Since point was to add open text proposals, why adding such mock-ish proposal type to replace in tests:
```rust
        .propose_detailed(
            "alice",
            "best proposal",
            "it's just the best",
            "do the thing",
            Proposal::DoTheThing {},
        )
```
where it looks like exactly something that should fit in `Text` (OpenText) type?
Or am I misunderstanding something?

Reply:
Actually a great point.

We should just test the Text proposal

Reply:
[One of the tests actually tests the `list_text_proposals` entrypoint, which filters out only text proposals out of all proposals](https://github.com/confio/poe-contracts/blob/3d879f7fc6a4af597f09574db2a7fa290b9329d9/packages/voting-contract/src/multitest/queries.rs#L348-L386), right? In tests I kind of want to make sure this filtering works, so I need some non-text proposal too, like in real contracts.

Reply:
Ah. So you don't store other data but iterate over all proposals.

We should expect the OC to have several hundred proposals in a few months (everyone wants their Engagement Points) and I think such an approach will not be efficient.

That is why I suggested just storing this data in it's own map. I guess the problem there is code duplication or a way to encapsulate it well to be imported?

Reply:
Okay, makes sense. I don't think this will prove too problematic. I didn't initially do it because I wasn't sure that was really how you wanted it + it felt weird to me to store the data twice, not for any serious concerns around structuring the code.

Reply:
Done!